.gitignore

```diff
@@ -1,14 +1,19 @@
 # See http://help.github.com/ignore-files/ for more about ignoring files.
 
 # compiled output
-/dist
-/tmp
-/out-tsc
+*/dist
+*dist
+*/tmp
+*/out-tsc
```

Comment:
I think these could be modified... originally, they said that anything in the root folder that matched should be ignored. Instead, it could be done with `dist` instead of `/dist` or `*/dist` ... You can also add .gitignore files to the folders such as

Reply:
Agreed. Most of the components will be ported over to the common library, and the assets should be as well. The hope is other duplicates to also be move to a singular shared source as well.